helm-chart/templates/_helpers.tpl

```diff
@@ -141,3 +139,33 @@ The name of the service account to use for backend pods
   value: "{{ .Release.Namespace }}"
 {{- end }}
 {{- end }}
+
+{{- define "cvat.sharedClickhouseEnv" }}
```

Comment:
Why not just merge this into `sharedBackendEnv`?

Reply:
These variables are needed only for some containers, I don't think it makes sense to specify it for all backed containers.